TreasureData/TreasureData.m

```diff
@@ -394,6 +413,18 @@ - (void)disableServerSideUploadTimestamp {
     self.serverSideUploadTimestampColumn = nil;
 }
 
+- (void)enableAutoAppendRecordUUID {
+    self.autoAppendRecordUUIDColumn = @"record_uuid";
+}
+
+- (void)enableAutoAppendRecordUUID: (NSString*)columnName {
+    self.autoAppendRecordUUIDColumn = columnName;
```

Comment:
This pointer of NSString is used also as a flag switching whether SDK should append a record UUID or not. If it's `nil`, a record UUID won't be appended. So if user passes `nil` here explicitly, it's just the same as calling `disableAutoAppendRecordUUID`.